Doc/library/othergui.rst

```diff
@@ -30,8 +30,11 @@ available for Python:
       for generating bindings for C++ libraries as Python classes, and
       is specifically designed for Python.
 
-   `PySide <https://wiki.qt.io/PySide>`_
-      PySide is a newer binding to the Qt toolkit, provided by Nokia.
+   `PySide2 <https://wiki.qt.io/PySide>`_
+      Also known as the Qt for Python project, PySide2 is a newer binding to the
+      Qt toolkit.
+      It is provided by The Qt Company and aims to provide a complete port of
+      PySide to Qt 5.
       Compared to PyQt, its licensing scheme is friendlier to non-open source
       applications.
```

Comment:
this paragraph looks like it needs to be rewrapped

Reply:
I wasn't sure about that one. Will do.